File: app/src/main/java/haivo/us/scanmusicpro/model/xml/Pitch.java
package haivo.us.scanmusicpro.model.xml;

import com.tickaroo.tikxml.annotation.PropertyElement;
import com.tickaroo.tikxml.annotation.Xml;

@Xml
public class Pitch {
    @PropertyElement
    String step;

    @PropertyElement
    String alter;

    @PropertyElement
    String octave;

    public String getStep() {
        return step;
    }
    public String getAlter() {
        return alter;
    }
    public String getOctave() {
        return octave;
    }
}

File: app/src/main/java/haivo/us/scanmusicpro/model/xml/Pitch.kt
package haivo.us.scanmusicpro.model.xml

import com.tickaroo.tikxml.annotation.PropertyElement
import com.tickaroo.tikxml.annotation.Xml

@Xml
data class Pitch(
        @PropertyElement
        val step: String?,

        @PropertyElement
        val alter: String?,

        @PropertyElement
        val octave: String?
)